python/ray/tune/experiment.py

```diff
@@ -105,6 +111,12 @@ def from_json(cls, name, spec):
         if "run" not in spec:
             raise TuneError("No trainable specified!")
 
+        if "repeat" in spec:
+            raise DeprecationWarning("The parameter `repeat` is deprecated; converting to `num_samples`. \
+                `repeat` will be removed in future versions of Ray.")```
```

Comment:
There's a typo at the end of this.

Also, I think this is over 80 chars - run `flake8 --ignore=C408,E121,E123,E126,E226,E24,E704,W503,W504` on `tune/` to make sure you're passing lint.